app/src/main/java/mihon/core/migration/Migration.kt

```diff
@@ -0,0 +1,19 @@
+package mihon.core.migration
+
+interface Migration {
+    val version: Float
+
+    suspend fun action(migrationContext: MigrationContext): Boolean
```

Comment:
Thoughts on making this invoke operator fucntion so we can do `migration(this@migrate)`?